docs/product/ai-in-sentry/sentry-prevent-ai/index.mdx

```diff
@@ -30,22 +30,32 @@ Once you have added a comment, the assistant will reply, acknowledging the reque
 
 ## Frequently Asked Questions
 
-**What data does Sentry Prevent need access to for the AI system to function, and what information is sent to third-party AI providers?**
+### What data does Sentry Prevent need access to for the AI system to function, and what information is sent to third-party AI providers?
 
-Sentry Prevent requires access to your pull requests, including PR metadata, repository information, file names, directory structures, and code diffs. The file names, code diffs, and PR descriptions are sent to the AI provider for analysis.
+Sentry Prevent requires access to your pull requests, including:
 
-**Does Sentry Prevent run anywhere outside of GitHub, and does it run in the background?**
+- PR metadata
+- Repository information
+- File names
+- Directory structures
+- Code diffs
 
-Sentry Prevent AI only runs on GitHub, and only runs when triggered by a comment. 
+The following are sent to the AI provider for analysis:
 
-The [Show Generative AI Features toggle](https://sentry.io/orgredirect/organizations/:orgslug/settings/#hideAiFeatures) does not currently apply to Sentry Prevent AI. if you don't want Sentry Prevent AI used on your projects, you can uninstall the [Seer app](https://github.com/apps/seer-by-sentry) at any time.
+- File names
+- Code diffs
+- PR descriptions
 
-You can learn more about AI privacy and security [here](/product/ai-in-sentry/ai-privacy-and-security/).
+### Does Sentry Prevent run anywhere outside of GitHub, and does it run in the background?
 
-## Feedback & Support
+No — Sentry Prevent AI **only runs on GitHub**, and only when explicitly triggered by a comment.
 
-Have feedback? Drop us a line on GitHub Issues or contact Sentry support.
```

Comment:
I took this out since it didn't link out and the module at the bottom each page covers the same ground.